tools/myst/myst.edl

```diff
@@ -513,5 +513,13 @@ enclave
             [out] uint32_t* rbx,
             [out] uint32_t* rcx,
             [out] uint32_t* rdx);
+        
+        long myst_get_mempolicy_ocall(
+            [out] int* mode,
+            [out] unsigned long* nodemask,
+            unsigned long maxnode,
+            [out] void* addr,
```

Comment:
Changed the configuration in the edl, this also needed me to either 
i)nodemask/=8 which is what I did, or 
ii) introduce a new parameter in the ocall which I wanted to avoid to keep it in line with the prototype in the linux manpage

let me know if you think option ii) is better in order to pass the size to the ocall

Reply:
As @bodzhang pointed out, the `addr` seems to be an input (not sure if there is in/out case, please double check). Also, the syscall only uses the `addr` as an address value instead of a pointer to a memory buffer, you can simply remove the `[]` annotation.